dutch/padel/padel.md

```diff
@@ -0,0 +1,5 @@
+Padel historie
+
+Padel is kort gezegd een mix tussen tennis en squash. Het ontstond in 1969, in de Mexicaanse stad Acapulco. Enrique Corcuera, een rijke Mexicaanse zakenman, veranderde de tennisregels, de tennisrackets en de baanafmetingen en bedacht hiermee een nieuwe sport: Padel.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.